index.html

```diff
@@ -866,12 +982,12 @@ <h4 class="modal-title">Help</h4>
                         scene.add(piece);
                     }
                     for (i = 0; i < this.totcaps; i++) {
-                        var stackno = Math.floor(this.tottiles / 10) + i;
+                        var stackno = Math.ceil(this.tottiles / 10) + i;
```

Comment:
This was a bug (I assume) where the Capstone was placed inside a tile when the 5x5 board was loaded.